server/graphql/v2/query/ExpensesQuery.ts

```diff
@@ -52,6 +56,26 @@ const ExpensesQuery = {
       where['CollectiveId'] = collective.id;
     }
 
+    // Add search filter
+    let include;
+    if (args.searchTerm) {
+      where[Op.or] = [
+        { description: { [Op.like]: `%${args.searchTerm}%` }},
```

Comment:
Potential SQL injection here, what if my `searchTerm` is something like:

```es6
searchTerm = `_' OR whateverSQLYoudLikeToInject OR ' `
```

Reply:
@Betree : Since this was not a [raw query](https://github.com/sequelize/sequelize/issues/6991) my understanding was that it's not subjected to sql injection. Is it not the case? I've tried things like `OR ... OR` and doesn't seem to have a problem. Do you mean we sanitize the string with a custom function before passing this in? :thinking:

Reply:
You're right, sequelize seems to sanitize what's passed to `searchTerm`.

However there's still a problem: if you pass a `%` character, it will be interpreted as wildcard and we want to prevent that.

Reply:
@Betree : Thanks for the review. I've changed it to use `Op.substring` which I think is better and removes the necessity to use any wildcard. :smile:

Reply:
Fantastic!